devlab/tests/generate_load.py

```diff
@@ -563,8 +563,12 @@ def clean_router_ports(router_id):
         def delete_user_keypairs(_user):
             if not _user.get('enabled'):
                 return
-            self.switch_user(user=_user['name'], password=_user['password'],
-                             tenant=_user['tenant'])
+            try:
+                self.switch_user(user=_user['name'],
+                                 password=_user['password'],
+                                 tenant=_user['tenant'])
+            except Exception:
```

Comment:
Catching `Exception` is considered bad practice, can you make exception handling here more narrow? Also, please add `log` or `print`, so that at least the exception is analyzable. 

Simple example of why it's bad idea to catch `Exception`: in case `_user` does not have `'tenant'` key for whatever reason (someone modified `config.py` and forgot about it), in current code you'll simply ignore that error, and will get further through the code and finally get even more cryptic error message, or (which is even worse) invalid results.